ansible_collections/arista/avd/examples/cv-pathfinder/intended/structured_configs/site2-wan2.yml

```diff
@@ -282,6 +282,7 @@ management_security:
       file: STUN-DTLS.crt
       key: STUN-DTLS.key
 metadata:
+  platform: vEOS
```

Comment:
IMO this should be CloudEOS since we currently combine VEOS and VEOS-LAB in our default platform_settings. CloudEOS is not there Today but it should probably be added together with the other WAN platforms.

Reply:
Actually, the reported version is now vEOS!

Reply:
Updated to CloudEOS